scripts/mocha.sh

```diff
@@ -2,11 +2,8 @@
 
 set -e -x
 
-for file in "$@"
-  do
-    mocha \
-      --require source-map-support/register \
-      -r should \
-      --compilers coffee:coffee-script/register,js:babel-core/register \
-      -t 30000 $file
-  done
+mocha \
+  --require source-map-support/register \
+  -r should \
+  --compilers coffee:coffee-script/register,js:babel-core/register \
+  -t 30000 $@
```

Comment:
Did some quick and dirty benchmarks. This provides more than a 10% increase on the two files I tried.